src/index.html

```diff
@@ -12,7 +12,72 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <section class="stars stars--0">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </section>
+    <section class="stars stars--1">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </section>
+    <section class="stars stars--2">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </section>
+    <section class="stars stars--3">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </section>
+    <section class="stars stars--4">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </section>
+    <section class="stars stars--5">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
```

Comment:
Consider adding empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. However, do not add them between parent and child elements.